tests/gtest/avifavmtest.cc

```diff
@@ -75,8 +75,10 @@ INSTANTIATE_TEST_SUITE_P(Tiny, AvmTest,
                                  Values(AVIF_PIXEL_FORMAT_YUV444),
                                  /*alpha=*/Values(false)));
 
+// width was originally 2. Increased it to 5 to work around
+// https://gitlab.com/AOMediaCodec/avm/-/issues/717.
```

Comment:
I do not think necessary to keep that comment. A few pixels are not representative of the performance of a codec. The test image is very small to make the test fast, but the content does not matter much, and any PSNR above 20 or 30 is fine in my opinion. Thanks for investigating.

Reply:
If we decrease `width` from 5 to 4 in the `HighBitDepthAndEvenDimensions` tests, the PSNR drops from 40.0 to 28.4 ~ 30.5 between the ["last known good commit"](https://gitlab.com/AOMediaCodec/avm/-/commit/c2f7e2f23f14b56c6a91dba364911603a645940d) and `research-v10.0.0`. I cannot dismiss a PSNR drop of this magnitude, even though the frame size is small (4x34). That's why I reported this issue to the AVM team.